cmd/virt-launcher/node-labeller/node-labeller.sh

```diff
@@ -32,6 +32,10 @@ if [ -e /dev/sev ]; then
   chmod o+rw /dev/sev
 fi
 
+if [ -e /var/lib/kubevirt-node-labeller/setcap-dac-override-enable ]; then
+  setcap cap_dac_override=ep /usr/libexec/qemu-kvm
```

Comment:
IMHO, we should not hardcode `qemu-kvm`. KubeVirt is also run in customized containers, where qemu binary location may differ, e.g. `/usr/bin/qemu-system-<arch>`. qemu-kvm is valid for distros like centos, fedora, rhel, but having it here will break compatibility with the builds from other vendors.